examples/07-Train-an-xgboost-model-using-the-Merlin-Models-API.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>My $0.02 is to remove the parens--they tend to diminish the importance of the words and what you have is no less important than the rest of the sentence.</p><p> </p><p>I'd also break the second sentence into two:</p><p> </p><p>"The dataset consists of <code>userId</code> and <code>movieId</code> pairings. For each record, a user rates a movie and the record includes additional information such as genre of the movie, age of the user, and so on."  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/522/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>